src/mds/MDSRank.cc

```diff
@@ -2835,7 +2835,7 @@ void MDSRankDispatcher::handle_asok_command(
     command_dump_tree(cmdmap, *css, f);
   } else if (command == "dump loads") {
     std::lock_guard l(mds_lock);
-    r = balancer->dump_loads(f);
+    r = balancer->dump_loads(f, cmdmap);
```

Comment:
please handle the cmdmap parsing and error reporting here and pass in only the `int` value of the `depth` to `dump_loads()` function
a default depth of `-1` can be assumed for infinite depth